core/src/main/resources/AppConstants.properties

```diff
@@ -415,26 +415,8 @@ ibistesttool.directory=${log.dir}/testtool
 messageLog.count.show=true
 errorStore.count.show=true
 
-#
-# Definitions for the FileViewerServlet
-#
-
-# a comma-separated list of words that will be marked
-FileViewerServlet.signal=ERROR,WARN,INFO,DEBUG
-
-FileViewerServlet.signal.ERROR.pre=<font color="red">
-FileViewerServlet.signal.ERROR.post=</font>
-FileViewerServlet.signal.WARN.pre=<font color="orange">
-FileViewerServlet.signal.WARN.post=</font>
-FileViewerServlet.signal.INFO.pre=<font color="blue">
-FileViewerServlet.signal.INFO.post=</font>
-FileViewerServlet.signal.DEBUG.pre=<font color="green">
-FileViewerServlet.signal.DEBUG.post=</font>
-
-# indication of stylesheet to use
-FileViewerServlet.signal.css=ie4.css
-
-FileViewerServlet.permission.rules=* * allow
+# Definitions for the FileViewer
+FileViewer.permission.rules=* * allow
```

Comment:
Er zijn mensen die nu de rechten hebben afgeschermd met de huidige property. Door deze nu te veranderen zet je effectief alles open wanneer men de naam niet aanpast bij een upgrade. Misschien is het het handigst om nog even de oude naam te gebruiken.

Reply:
Misschien is het slim om beide opties aan te bieden?

Reply:
Dan kan je `FileViewer.permission.rules=${FileViewerServlet.permission.rules}` gebruiken 😄

Reply:
> Dan kan je `FileViewer.permission.rules=${FileViewerServlet.permission.rules}` gebruiken 😄

Ah en hiermee hoef ik alleen binnen de code naar `FileViewer.permission.rules` te kijken toch?
Had een check voor beide toegevoegd maar dit zou wel beter zijn